test/models/user_test.rb

```diff
@@ -31,25 +30,25 @@ def setup
     assert_not @user.valid?
   end
 
-  test "email validation should accept valid addresses" do
-    valid_addresses = %w[user@example.com USER@foo.COM A_US-ER@foo.bar.org
-                         first.last@foo.jp alice+bob@baz.cn]
+  test 'email validation should accept valid addresses' do
+    valid_addresses = %w(user@example.com USER@foo.COM A_US-ER@foo.bar.org
+                         first.last@foo.jp alice+bob@baz.cn)
     valid_addresses.each do |valid_address|
       @user.email = valid_address
       assert @user.valid?, "#{valid_address.inspect} should be valid"
     end
   end
 
-  test "email validation should reject invalid addresses" do
-    invalid_addresses = %w[user@example,com user_at_foo.org user.name@example.
-                           foo@bar_baz.com foo@bar+baz.com]
+  test 'email validation should reject invalid addresses' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.